server/src/main/java/org/apache/calcite/sql/ddl/SqlDropFunction.java

```diff
@@ -25,7 +27,8 @@
 /**
  * Parse tree for {@code DROP FUNCTION} statement.
  */
-public class SqlDropFunction extends SqlDropObject {
+public class SqlDropFunction extends SqlDropObject
+    implements SqlExecutableStatement {
```

Comment:
isn't `SqlDropObject` already implement `SqlExecutableStatement` ?

Reply:
It is, but does not include `create/drop` function, we can remove the implements from `SqlExecutableStatement`.